src/Resources/config/services/field_types.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="true" />
```

Comment:
Should this be public?

Reply:
In original `vendor/sylius/grid-bundle/src/Bundle/Resources/config/services/field_types.xml` we have `<defaults public="true" />` too, but I've tried and looks like it works without that